project.clj

```diff
@@ -78,6 +78,9 @@
                                             (some->> x :ns ns-name str (re-matches (re-pattern (apply str patterns)))))))
                    :generative (fn [x] (some->> x :ns ns-name str (re-matches #"^clara\.generative.*")))
                    :performance (fn [x] (some->> x :ns ns-name str (re-matches #"^clara\.performance.*")))}
+
+  ;; Remove any "Test Facts" from being included in the jar.
+  :jar-exclusions [#".*TestFact\.(java|class)"]
```

Comment:
They really shouldn't even be in the "main" src path, but that's a topic for another build related issue I think.

Any reason this was added in this particular PR though?

Reply:
removed with:
https://github.com/cerner/clara-rules/pull/449/commits/b41ffee72a58f19e94b2169f4e9b7ed35736551c